test/runtests.jl

```diff
@@ -37,4 +37,5 @@ include("nlp.jl")
 include("generate_and_solve.jl")
 include("print.jl")
 include("operator.jl")
-include("macros.jl")
\ No newline at end of file
+include("macros.jl")
+include("hygiene.jl")
```

Comment:
The reason hygiene was run separately from the rest of the tests was to make sure it worked when JuMP hadn't already been loaded using `using`. Can we emulate that somehow?

Reply:
Maybe with another job like @odow did in https://github.com/JuliaOpt/JuMP.jl/pull/1485 ?